app/test_main.py

```diff
@@ -1,3 +1,21 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_two_parameters_equal_to_zero() -> None:
+    assert (get_human_age(0, 0) == [0, 0])
```

Comment:
The test case for `test_two_parameters_equal_to_zero` seems to be correct. However, ensure that this case is not repeated in other tests to adhere to the 'Don't Repeat Yourself' principle.